java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -155,25 +155,31 @@ class ReferenceModeStore private constructor(
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     val backingStore = DirectStoreMuxer<CrdtData, CrdtOperation, Any?>(
         storageKey = backingKey,
-        backingType = backingType,
-        callbackFactory = { muxId ->
-            ProxyCallback { message ->
-                CoroutineScope(coroutineContext).launch {
-                    receiveQueue.enqueue(
-                        PreEnqueuedFromBackingStore(message.toReferenceModeMessage(), muxId)
-                    )
-                }
-            }
-        }
+        backingType = backingType
     )
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
I don't think we can use VisibleForTesting in `arcs/core`? I'd be curious if this still compiles to JS safely.

Reply:
I've removed VisibleForTesting and left it as public